### ISSUE TITLE
Default Airpopup not dissapearing when being disabled.

### ISSUE THREAD

**Issue by TomHarro** [vanwijkdave/Ainsworth-Issues#37]
After creating custom popup, and disabling the default one, the custom one appears, with the old popup appearing underneath it.

palera1n v2, beta 7, palen1x 1.1 rootful, ios 15.5b4.

**Comment by vanwijkdave** [comment]
Can you maybe upload a screen capture of this issue? I am not completely sure that I understand the full situation.

**Comment by TomHarro** [comment]
https://github.com/vanwijkdave/Ainsworth-Issues/assets/88755366/ff3472d3-cecf-47f9-b305-c0610cfd647f

Either the popup appears first or last but it’s not always disabled.

**Comment by vanwijkdave** [comment]
I see you closed this issue, is it resolved?

**Comment by TomHarro** [comment]
No, I don't know why it closed, maybe I did it accidentally. But its not resolved.

**Comment by vanwijkdave** [comment]
Hi, a few questions to get a better idea of the issue:
- Does this always happen?
- Is the toggle in the first screen in settings about the proximity popup ON?
- Which AirPods are these?
- Is any tweak related to bluetooth or AirPods installed?

**Comment by TomHarro** [comment]
Does this always happen?
No, sometimes it works fine, with the airpods popup not appearing.
Is the toggle in the first screen in settings about the proximity popup ON?
idk where to find this setting, so idk.
Which AirPods are these?
Pro, Gen 1.
Is any tweak related to bluetooth or AirPods installed?
No.

**Comment by TomHarro** [comment]
i didnt mean to close the thread.
